osmoutils/accum/accum_helpers.go

```diff
@@ -6,10 +6,6 @@ import (
 	"github.com/osmosis-labs/osmosis/osmoutils"
 )
 
-var (
-	minusOne = sdk.NewDec(-1)
```

Comment:
huh, why didn't this get called out by the unused or deadcode linters

Reply:
ikr, odd